charts/nri-bundle/README.md

```diff
@@ -1,11 +1,32 @@
 # nri-bundle
 
-![Version: 4.5.0](https://img.shields.io/badge/Version-4.5.0-informational?style=flat-square)
+![Version: 4.5.6](https://img.shields.io/badge/Version-4.5.6-informational?style=flat-square)
 
-A chart which groups together the individual charts for the New Relic Kubernetes solution for more comfortable deployment.
+Groups together the individual charts for the New Relic Kubernetes solution for a more comfortable deployment.
 
 **Homepage:** <https://github.com/newrelic/helm-charts>
 
+## Bundled charts
+
+This chart do not deploy anything by itself but has many charts as dependencies install and upgrade the New Relic Kubernetes
+Integration using only one chart.
+
+In case you need more information of each component this chart installs or you are and advanced user that want to install each
+component separated, here is a list of components that this chart installs and where you can have more information about them:
+
+| Component                    | Installed by default? | Description |
+|------------------------------|-----------------------|-------------|
+| [newrelic-infrastructure](https://github.com/newrelic/nri-kubernetes/tree/main/charts/newrelic-infrastructure) | Yes | This component will (in privileged mode) monitor the nodes and run integrations to monitor services inside the Kubernetes cluster (in any mode). It will also deploy a daemonset in the control plane nodes to give information about the health of the control plane and deployment to scrape the data from the KSM (see below) |
+| [nri-metadata-injection](https://github.com/newrelic/k8s-metadata-injection/tree/main/charts/nri-metadata-injection) | Yes | Webhook that will inject environments variables to all the pods so the APM agents have the context to know where they are running. |
+| [kube-state-metrics](https://github.com/kubernetes/kube-state-metrics/tree/master/charts/kube-state-metrics) | | kube-state-metrics (KSM) listens to the Kubernetes API server and generates metrics about the state of the objects. It is not focused on the health of the individual Kubernetes components, but on the health of the various objects inside, such as deployments and pods. |
+| [nri-kube-events](https://github.com/newrelic/nri-kube-events/tree/main/charts/nri-kube-events) | | This component is an active watcher of event resource in the Kubernetes system that take those infrastructure events (like a pod has been created or a pod have just crashed) and pushes them to New Relic. |
+| [newrelic-infra-operator](https://github.com/newrelic/newrelic-infra-operator/tree/main/charts/newrelic-infra-operator) | | (Beta) This operator is meant to be used with Fargate clusters or serverless clusters that need to deploy the infrastructure agent as a sidecar per pod. |
```

Comment:
```suggestion
| [newrelic-infra-operator](https://github.com/newrelic/newrelic-infra-operator/tree/main/charts/newrelic-infra-operator) | | (Beta) Used with Fargate or serverless environments to inject `newrelic-infrastructure` as a sidecar instead of the usual DaemonSet. |
```